flang/include/flang/Optimizer/Support/TypeCode.h

```diff
@@ -34,22 +34,21 @@ inline int characterBitsToTypeCode(unsigned bits) {
   // clang-format on
 }
 
-inline int complexBitsToTypeCode(unsigned bits) {
+inline int complexBitsToTypeCode(unsigned bitwidth) {
   // clang-format off
-  switch (bits) {
-  case 16:  return CFI_type_half_float_Complex; // CFI_type_bfloat_Complex ?
+  switch (bitwidth) {
   case 32:  return CFI_type_float_Complex;
   case 64:  return CFI_type_double_Complex;
-  case 80:  return CFI_type_extended_double_Complex;
-  case 128: return CFI_type_float128_Complex;
+  case 80:
+  case 128: return CFI_type_long_double_Complex;
```

Comment:
This appears to be backwards. Didn't @vdonaldson just add the new CFI codes?

Reply:
Was not there when this was upstreamed I guess. This sync is from smth upstreamed back in November or December.

Reply:
Should we use `CFI_type_extended_double_Complex` for `case 80:`?

Reply:
Ok it's from e43b2e4f480b507a1d3eb9a5f3b294c0599eed73. It was not there before. I'll update this PR and then upstream the change back to main.

Reply:
@schweitzpgi I just realized that I did a really bad sync at first. I didn't noticed the changes and just updated it so it should be just a name change right now. It was asked by the reviewer...

Reply:
Looks ok to me